build.gradle

```diff
@@ -18,22 +18,20 @@ buildscript {
     repositories {
         jcenter()
         mavenCentral()
+        google()
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:2.3.3"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.0"
-        classpath "com.jakewharton:butterknife-gradle-plugin:8.7.0"
+        classpath "com.android.tools.build:gradle:3.2.0-alpha05"
+        classpath "com.jakewharton:butterknife-gradle-plugin:8.8.1"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.2.21"
```

Comment:
The newest version is 1.2.30. However, Android Studio 3.2.0 includes Kotlin plugin v1.2.21.